data/RecordAction.js

```diff
@@ -5,6 +5,35 @@
  * Copyright © 2018 Extremely Heavy Industries Inc.
  */
 
+import {Icon} from '@xh/hoist/icon';
+import {isBoolean, isNumber, isNil} from 'lodash';
+
+export const addAction = {
```

Comment:
There are interesting.  More graphical, than anything, as they don't actually have actionFns.

Do we think this is really generalizable as a hoist API? Perhaps they should just be defined in the Rest package, where we actually have implementations?